Dashboards/Images/Logos/ThycoticLogo.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.2" baseProfile="tiny" id="Слой_1"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 39 39"
+	 xml:space="preserve">
+<circle fill="#081F2D" cx="19.5" cy="19.5" r="19.3"/>
+<linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="15.026" y1="10.0484" x2="24.5654" y2="19.5878">
+	<stop  offset="0" style="stop-color:#80BE41"/>
```

Comment:
Invalid logo.
Ensure raw file of logo does not have any of the following:
style formats
xmlns:xlink
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Please refer https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md

Reply:
Change Logo file